app/experimenter/nimbus-ui/src/components/PageNew/index.tsx

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useCallback } from "react";
+import { RouteComponentProps } from "@reach/router";
+import AppLayout from "../AppLayout";
+import LinkExternal from "../LinkExternal";
+import FormExperimentOverviewPartial from "../FormExperimentOverviewPartial";
+
+const TRAINING_DOC_URL =
+  "https://mana.mozilla.org/wiki/display/FJT/Project+Nimbus";
+
+type PageNewProps = {} & RouteComponentProps;
+
+const PageNew = (props: PageNewProps) => {
+  // TODO: Get this from constants / config loaded at app start?
```

Comment:
You also got us started on EXP-462! I'm tempted to bump that issue down to a 3 now, heh.

We can address this there, the check looks cut off on the right-hand side.

<img width="1380" alt="image" src="https://user-images.githubusercontent.com/13018240/96276660-eb2d6b00-0f98-11eb-9cf3-9a0b4167e268.png">